tests/functional/r/regression_02/regression_distutil_import_error_73.py

```diff
@@ -7,7 +7,7 @@
 https://github.com/PyCQA/astroid/pull/1321
 """
 
-# pylint: disable=unused-import
+# pylint: disable=unused-import, deprecated-module
```

Comment:
Shall we add the message to expected messages here? Otherwise there is no test for the `distutils` deprecation warning.

Reply:
Sure thing -- it will require duplicating this file so that one runs with `max_pyver` and one with `min_pyver`. Just wanted to make sure that was the direction to go.

Reply:
Hm, then maybe do a separate test file? Focus on the `deprecated-module` warning and keep this with the disable?